qiita_pet/templates/study_ajax/base_info.html

```diff
@@ -0,0 +1,7 @@
+<p style="font-weight:bold;">Abstract</p>
+<p>{{study_info['study_abstract']}}</p>
+Study ID:  {{study_info['study_id']}}<br />
+Publications: {% for pub in study_info['publication_doi'] %}<a href="http://dx.doi.org/{{pub}}" target="_blank">{{pub}}</a> {% end %}<br />
```

Comment:
This formatter already exist in other sections of the code, not sure if it will be better to have that for loop and the formatter outside the template and simply print the value here, what do you think?